suez.py

```diff
@@ -31,7 +31,8 @@ def _since(ts):
     help="Type of LN client.",
 )
 @click.option(
-    "--client-args", default="", help="Extra arguments to pass to client RPC."
+    "--client-args", default=("",""), multiple=True,
```

Comment:
`default=("",""),`

Doesn't this evaluate to True in the `_run` function? 

``` python
        if self.client_args:
             args = ["lightning-cli"] + list(self.client_args) + list(args)
```

where it passes two empty arguments?

Reply:
yes, thanks. Passing `default=[]` instead solves this in https://github.com/prusnak/suez/pull/15/commits/190ada56eaf89b05cd719a7c9b08fd1d7f687556